src/db/mongodb/collection.rs

```diff
@@ -111,6 +112,13 @@ pub trait MongoDbCollectionExt: MongoDbCollection {
     ) -> Result<UpdateResult, Error> {
         self.with_type().replace_one(query, replacement, options).await
     }
+
+    /// Returns the number of documents in the collection.
+    #[cfg(feature = "rand")]
```

Comment:
Maybe we can leave out the feature gate here.

Reply:
We could, but atm we have no other use-case for that functionality except for checking correctness of collection inserts.

Reply:
alright